### PR TITLE
Add lite server

### PR DESCRIPTION
**Pull request by magmat88**
Added files with styles to app and components, imported styles in components. Added lite server.

### CODE REVIEW

**Review by earendillo — APPROVED** [review]

File: src/app/app.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
+// Variables
+@import './styles/variables/variables.css';
```
Comment (earendillo) [inline src/app/app.scss:2]: why do you import .css files in .scss?

File: src/app/app.tsx [hunk 2]
```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { StoreContext, useStoreon } from 'storeon/react';
-import './app.scss';
 import { AppStore } from './app.store';
-import { Report } from './modules/reports/reports.state';
 import { Link, Router, View } from 'react-navi';
 import { compose, mount, route, withView } from 'navi';
-import { Articles } from './components/articles.component';
-import { Reports } from './components/reports.component';
+// Components
```
Comment (earendillo) [inline src/app/app.tsx:6]: comments for imports are redundant; please remove them

File: src/app/components/articles/articles.component.tsx [hunk 3]
```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import { useStoreon } from 'storeon/react';
-import { Article } from '../modules/articles/articles.state';
+// Modules
```
Comment (earendillo) [inline src/app/components/articles/articles.component.tsx:3]: same, comments for imports should be removed

File: src/app/styles/variables/colors.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+$color_link-hover: $color_contrast-hover;
+
+$color_border-button: #ccc;
+$color_border-button-hover: #aaa;
+$color_border-button-focus: #bbb;
+$color_border1: #e1e1e1;
+$color_border2: #666;
+
+
+// $color__border-button: #ccc #ccc #bbb;
```
Comment (earendillo) [inline src/app/styles/variables/colors.scss:24]: remove or uncomment; we do not commit commented code

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/app.tsx (hunk 2 above)
Comment (magmat88) [reply src/app/app.tsx]: Comments for imports removed.

File: src/app/app.scss (hunk 1 above)
Comment (magmat88) [reply src/app/app.scss]: Files in imports changed from .css to .scss.

File: src/app/components/articles/articles.component.tsx (hunk 3 above)
Comment (magmat88) [reply src/app/components/articles/articles.component.tsx]: Comments for imports removed.

File: src/app/styles/variables/colors.scss (hunk 4 above)
Comment (magmat88) [reply src/app/styles/variables/colors.scss]: Unnecessary commented code removed.

**Pull request merged**